src/arch/riscv/irqc/aia/inc/imsic.h

```diff
@@ -0,0 +1,108 @@
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Bao Project and Contributors. All rights reserved.
+ */
+
+#ifndef IMSIC_H
+#define IMSIC_H
+
+#include <bao.h>
+#include <platform.h>
+
+#define IMSIC_MAX_INTERRUPTS (PLAT_IMSIC_MAX_INTERRUPTS)
+/** We only support 1 guest per hart at the moment */
+#define IMSIC_NUM_VS_FILES   (1)
+#define IMSIC_NUM_FILES      (IMSIC_NUM_VS_FILES + 1)
+
+#define STOPEI_EEID          (16)
+
+#define IMSIC_EIDELIVERY     (0x70)
+#define IMSIC_EITHRESHOLD    (0x72)
+#define IMSIC_EIP            (0x80)
+#define IMSIC_EIE            (0xC0)
+
+struct imsic_intp_file_hw {
+    uint32_t seteipnum_le;
+    uint32_t seteipnum_be;
+} __attribute__((__packed__, aligned(0x1000ULL)));
+
+struct imsic_global_hw {
+    struct imsic_intp_file_hw s_file;
+} __attribute__((__packed__, aligned(0x1000ULL)));
+
+/**
+ * @brief Initilaizes the IMSIC
```

Comment:
typo in Initializes

Reply:
Done. Take a look [here](https://github.com/D3boker1/bao-hypervisor/blob/bd10f0347eefad0e3f21652d51d09c8eb40979be/src/arch/riscv/irqc/aia/inc/imsic.h#L34)